public/app/features/explore/Explore.tsx

```diff
@@ -71,8 +73,15 @@ const getStyles = stylesFactory(() => {
       margin-top: 21px;
     `,
     button: css`
+      label: button;
       margin: 1em 4px 0 0;
     `,
+    queryContainer: css`
+      label: queryContainer;
+      // Need to override normal css class and don't want to count on ordering of the classes in html.
+      height: auto !important;
+      padding: ${theme.panelPadding}px;
```

Comment:
if you want to not have to extend & override panel-container, you could just add background: ${theme.colors.panelBg}  and same for border

Reply:
Going to leave it using the panel-container. We already use it for the graph/table panels in explore so probably better that is in sync.